resources/Microsoft.DSC.Experimental/0.0.1/Microsoft.DSC.Experimental.psd1

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+@{
+
+# Script module or binary module file associated with this manifest.
+RootModule = 'Microsoft.DSC.Experimental.psm1'
```

Comment:
We may have more experimental resources in the future, you should call this `Microsoft.DSC.Experimental.Systemctl`

Reply:
renamed.